gemsupervisor/xml/gemsupervisor_tif_lemo.xml

```diff
@@ -138,18 +138,6 @@
     <xc:Application class="gem::supervisor::GEMSupervisor" id="254" instance="0" network="local">
       <properties xmlns="urn:xdaq-application:GEMSupervisor"
 		  xsi:type="soapenc:Struct">
-        <DatabaseInfo xsi:type="soapenc:Struct">
-          <dbName xsi:type="xsd:string">ldqm_test_db</dbName>
-          <dbHost xsi:type="xsd:string">gem904daq01.cern.ch</dbHost>
-          <dbPort xsi:type="xsd:integer">3306</dbPort>
-          <dbUser xsi:type="xsd:string">gemdaq</dbUser>
-          <dbPass xsi:type="xsd:string">gemdaq</dbPass>
-          
-          <setupTag xsi:type="xsd:string">teststand</setupTag>
-          <runPeriod xsi:type="xsd:string">2016T</runPeriod>
-          <setupLocation xsi:type="xsd:string">TIF</setupLocation>
-        </DatabaseInfo>
-
 	<rcmsStateListener xsi:type="soapenc:Struct">
```

Comment:
To be honest, this file (and all others here not named `gemsupervisor.xml` probably should be removed...

Reply:
Ok, great! I was hesitating to remove them, but I'll remove them as part of my next cleanup PR (targeting at the removal of unused files).

In the future, the configuration files should probably be stored somewhere else (in the `GEMRunConfigs` repository?) or maintained up-to-date the main repository.

Reply:
Indeed, *actual* configs should be stored where you mention, but the example configs here should be kept to a minimum and stick with the source (and maybe at some point be automatically generated to stay up-to-date with potential code changes)